templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -66,3 +66,12 @@ jobs:
             Bump minor version
             [noissue]
           delete-branch: true
+
+      - name: add new backport label to the affected repository
+        run: |
+          curl -X POST \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: Bearer {{ '${{ secrets.RELEASE_TOKEN }}' }}"\
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/pulp/{{ plugin_name }}/labels \
+          -d '{"name":"backport-{{ "${{ github.event.inputs.name }}" }}","color":"d4c5f9"}'
```

Comment:
@gerrod3 I have seen that the backport labels are usually pastel colors, so I have chosen one since I didn't want to deal with something like "pastel color generator". Are you okay if I just make it a completely random color?